Basic/APIDiorama/Assets/Common/Scripts/com.unity.samples.api-diorama.asmdef

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "com.unity.samples.api-diorama",
+    "rootNamespace": "",
+    "references": [
+        "GUID:1491147abca9d7d4bb7105af628b223e",
```

Comment:
could we switch to string instead of GUIDs please? This'll make reviews easier. GUIDs are great at the beginning when setting up assemblies and renaming things all the time. but when those are established, strings just makes things easier to track :)

Reply:
Looks like a bad practice as renaming assemblies causes this to break, but I get your point. Done.